packages/core/src/index.ts

```diff
@@ -18,7 +18,15 @@ export {
   setUser,
   withScope,
 } from './exports';
-export { getCurrentHub, getHubFromCarrier, Hub, makeMain, getMainCarrier, setHubOnCarrier } from './hub';
+export {
+  getCurrentHub,
+  getHubFromCarrier,
+  Hub,
+  makeMain,
+  getMainCarrier,
+  setHubOnCarrier,
+  runWithAsyncContext,
```

Comment:
do we need to export `setAsyncContextStrategy`?

Reply:
Oh yes, good catch!